monstereo/visuals/printer.py

```diff
@@ -20,6 +23,75 @@ def get_angle(xx, zz):
     return angle
 
 
+def get_pifpaf_outputs(annotations):
```

Comment:
If you get circular imports, instead of duplicating the functions, can you directly move them in a different place?